src/app/public/modules/text-expand/text-expand-adapter.service.ts

```diff
@@ -11,20 +11,22 @@ export class SkyTextExpandAdapterService {
   private renderer: Renderer2;
 
   constructor(
-    private rendererFactory: RendererFactory2
+    rendererFactory: RendererFactory2
```

Comment:
Why are we removing the `private` keyword from this?

Reply:
There's no need for the `rendererFactory` to be a property of the class itself; it can be a local param for the constructor, which will reduce bundle size.